features/checkout/paying_for_order/preventing_to_change_payment_method_of_already_cancelled_order.feature

```diff
@@ -0,0 +1,30 @@
+@paying_for_order
+Feature: Preventing to change payment method of already cancelled order
+    In order to avoid invalid order data
+    As a Customer
+    I should be prevented from changing payment method of the cancelled order
+
+    Background:
+        Given the store operates on a single channel in "United States"
+        And the store has a product "Angel T-Shirt"
+        And the store ships everywhere for Free
+        And the store allows paying with "Cash on Delivery"
+        And the store also allows paying with "Bank Transfer"
+        And I am a logged in customer
+        And I placed an order "#00000666"
+        And I bought a single "Angel T-Shirt"
+        And I addressed it to "Lucifer Morningstar", "Seaside Fwy", "90802" "Los Angeles" in the "United States" with identical billing address
+        And I chose "Free" shipping method with "Cash on Delivery" payment
```

Comment:
I know that this is how we describe the background in other tests, but I believe it would be more in line with art to use the passive form
```suggestion
        And the customer is logged in
        And he placed an order "#00000666"
        And he bought a single "Angel T-Shirt"
        And he addressed it to "Lucifer Morningstar", "Seaside Fwy", "90802" "Los Angeles" in the "United States" with identical billing address
        And he chose "Free" shipping method with "Cash on Delivery" payment
```

Reply:
I'd prefer to stick to the convention we're using now. In the scenario, we play a customer role, so **I am** a customer.